src/templates/research-topic.js

```diff
@@ -162,44 +162,36 @@ const TopicNext = ({ pageData }) => {
       : pageData.researchItems[nextTopicIndex + 1]
   // console.log(nextTopic)
 
-  return (
-    <div className="topic-next">
-      <Container fluid="sm">
-        <Row>
-          <Col
-            xs={{ size: 12, offset: 0 }}
-            sm={{ size: 12, offset: 0 }}
-            md={{ size: 10, offset: 1 }}
-            lg={{ size: 10, offset: 1 }}
-            xl={{ size: 10, offset: 1 }}
-          >
-            <div
-              className="topic-next__card"
-              style={{ backgroundColor: nextTopic.item_color }}
-            >
-              <h5>Next topic</h5>
-              <h3>{nextTopic.label}</h3>
-              <p>{nextTopic.item_description}</p>
-              <Link
-                to={`/research-library/${slugify(nextTopic.label)}/`}
-                className="dotted-bottom"
-              >
-                Learn more
-              </Link>
-              {nextTopic.item_image && (
-                <div className="topic-next__card-image">
-                  <Image
-                    className="h-100 w-100"
-                    filename={nextTopic.item_image}
-                  />
-                </div>
-              )}
-            </div>
-          </Col>
-        </Row>
-      </Container>
-    </div>
-  )
+    return (
+        <div className="topic-next">
+            <Container fluid="sm">
+                <Row>
+                    <Col
+                        xs={{ size: 12, offset: 0 }}
+                        sm={{ size: 12, offset: 0 }}
+                        md={{ size: 10, offset: 1 }}
+                        lg={{ size: 10, offset: 1 }}
+                        xl={{ size: 10, offset: 1 }}
+                    >
+                        <div className="topic-next__card" style={{backgroundColor: nextTopic.item_color}}>
+                            <h2>Next topic</h2>
+                            <h3>{nextTopic.label}</h3>
+                            <p>{nextTopic.item_description}</p>
+                            <Link to={`/research-library/${slugify(nextTopic.label)}/`} className="dotted-bottom">Learn more</Link>
+                            {nextTopic.item_image && 
+                                <div className="topic-next__card-image">
+                                    <Image
+                                        className="h-100 w-100"
+                                        filename={nextTopic.item_image}
+                                    />
+                                </div>
+                            }
+                        </div>
+                    </Col>
+                </Row>
+            </Container>
+        </div>
+    )
```

Comment:
noticing switch to 4 character indent here.  are you using prettier to autoformat?  if so it doesn't seem to be picking up on the `.prettierrc`.